### ISSUE TITLE
Importing User and Passwords from older Version Mypal

### ISSUE THREAD

**Issue by IP-CAM** [Feodor2/Mypal68#37]
Could you please tell me the Name of the MyPal v.29.3 File, containing the User Access Data like Usernames and Passwords. I can see it within the Browser under Saved Logins, but I don't know it's File-Name and -Location within the Installation. I like to import the data into MyPal68, either manually, or then by use of some import-routine. Thank you.

**Comment by NoNameNeeded2** [comment]
It doesn't work.
I've tried it and copied all the files over to the new profile but it doesn't work.
In addition to that you won't be able to save any new passwords on Mypal68 if you move the old files to the new profile folder.
The files in question are key3.db and logins.json but it doesn't work.

It's also not possible to export the usernames and passwords from the old profile and to import them in Mypal68, because even though there are addons that allow you to export this data from the old version of Mypal and even though you can actually import passwords and usernames in Mypal68 from a file, the format in which this data is stored isn't compatible with the format that Mypal68 expects.

So, in summary it can be said that there seems to be no convenient way to import the passwords and user names in Mypal68.
So it's best to keep the old version of Mypal and manually copy passwords and user names if you need them.

**Comment by NoNameNeeded2** [comment]
key4.db and logins.json are the files in question

**Comment by IP-CAM** [comment]
Thank you for your (detailed first eMail) info on this, I make an Picture of all my logins, to eventually find my huge amount of Access Data there, whenever I access such Sites.

**Comment by NoNameNeeded2** [comment]
I deleted my first post because it turned out to be wrong.
It IS possible to transfer the password and user names data to the new profile.

Copying the two files from your old profile to your new profile (Mypal68) should do the trick.
(Make sure Mypal68 is closed though when you replace the two files)

**Comment by IP-CAM** [comment]
I will try ! Thanks a Lot!

**Comment by IP-CAM** [comment]
Seems to work perfect, thanks a lot again.
